new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -214,156 +224,281 @@ public void testCreateMiiDomain() {
     withStandardRetryPolicy
         .conditionEvaluationListener(
             condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
                     + "(elapsed time {2}ms, remaining time {3}ms)",
-                domainUid,
+                domainUID,
                 domainNamespace,
                 condition.getElapsedTimeInMS(),
                 condition.getRemainingTimeInMS()))
-        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+        .until(domainExists(domainUID, DOMAIN_VERSION, domainNamespace));
 
 
     // check admin server pod exist
     logger.info("Check for admin server pod {0} existence in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodCreated(adminServerPodName);
+    checkPodCreated(adminServerPodName, domainUID, domainNamespace);
 
     // check managed server pods exists
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Check for managed server pod {0} existence in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkPodCreated(managedServerPrefix + i);
+      checkPodCreated(managedServerPrefix + i, domainUID, domainNamespace);
     }
 
     // check admin server pod is running
     logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkPodRunning(adminServerPodName);
+    checkPodRunning(adminServerPodName, domainUID, domainNamespace);
 
     // check managed server pods are running
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkPodRunning(managedServerPrefix + i);
+      checkPodRunning(managedServerPrefix + i, domainUID, domainNamespace);
     }
 
     logger.info("Check admin service {0} is created in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkServiceCreated(adminServerPodName);
+    checkServiceCreated(adminServerPodName, domainNamespace);
 
     // check managed server services created
     for (int i = 1; i <= replicaCount; i++) {
       logger.info("Check managed server service {0} is created in namespace {1}",
           managedServerPrefix + i, domainNamespace);
-      checkServiceCreated(managedServerPrefix + i);
+      checkServiceCreated(managedServerPrefix + i, domainNamespace);
+    }
+
+  }
+
+  @Test
+  @Order(2)
+  @DisplayName("Create a second domain with the image from the the first test")
+  @Slow
+  @MustNotRunInParallel
+  public void testCreateMiiSecondDomain() {
+    // admin/managed server name here should match with model yaml in WDT_MODEL_FILE
+    final String adminServerPodName = domainUID1 + "-admin-server";
+    final String managedServerPrefix = domainUID1 + "-managed-server";
+    final int replicaCount = 2;
+
+    logger.info("Creating unique namespace for Domain domain2");
```

Comment:
Sankar has added injection of the namespaces in the initAll() method so that the diagnostic code can collect data of all namespaces that are used in the test. Once you sync with the latest develop branch, you can just increase the number of namespaces in the initAll() method; no need to create one yourself.

Reply:
yes, will do. Sankar's PR hadn't been merged when I created this PR. I will sync develop with my branch and make the changes.